supertokens_python/recipe/session/recipe_implementation.py

```diff
@@ -107,15 +104,24 @@ async def create_new_session(
             result["accessToken"]["token"]
         ).payload
 
+        refresh_token = result.get("refreshToken")
+        refresh_token_info: Optional[TokenInfo] = None
+        if refresh_token is not None:
```

Comment:
When could this be `None`?

Reply:
Shouldn't be. Unless something is wrong with the core response. 

We should throw error. So updated accordingly.